client/packages/system/src/Item/api/operations.graphql

```diff
@@ -77,11 +77,31 @@ fragment PackagingVariant on PackagingVariantNode {
   volumePerUnit
 }
 
+fragment BundledItemVariant on ItemVariantNode {
+  id
+  name
+  itemId
+  itemName
+}
+
+fragment BundledItem on BundledItemNode {
+  __typename
+  id
+  ratio
+  principalItemVariant {
```

Comment:
Are we being inconsistent with naming? Is it the "base" item or the "principal" item? Can we keep it the same throughout -- I think I have a slight preference for "base", but not a strong one.